CHANGELOG.rst

```diff
@@ -18,15 +18,22 @@
   if a test suite uses ``pytest_plugins`` to load internal plugins (`#1888`_).
   Thanks `@jaraco`_ for the report and `@nicoddemus`_ for the PR (`#1891`_).
 
+* Do not call tearDown (and cleanups) when running unittest with ``--pdb``
```

Comment:
I think we should make crystal clear that this change only affects unittest-tests, so I suggest replacing:

> Do not call tearDown (and cleanups) when running unittest with ``--pdb``

With:
 
> Do not call tearDown and cleanups when running tests from `unittest.TestCase` subclasses with ``--pdb``